test/e2e-upgrade-tests.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
```

Comment:
Is this copyright header up to date?

Reply:
Started in 2019 :-P

Reply:
let's keep it consistent with the boilerplate generated by `hack/boilerplate/boilerplate.go.txt` and regenerate the header to 2020